cfgov/unprocessed/apps/youth-employment-success/js/util.js

```diff
@@ -0,0 +1,110 @@
+let UNDEFINED;
+
+const REDUCER_RETURN_ERROR = 'Reducer must return a state object';
+const INVALID_ARG_ERROR = 'reducers argument must be an object, where each value is a reducer function';
+
+/**
+ *
+ * @param {object} state Composed of key / value pairs where the key is the name of
+ *  a slice of app state, and the value is the values for that slice of state
+ * @param {string} key name of the state slice
+ * @returns {function|undefined} Return the reducer function or undefined
+ */
+function checkForPreviousState( state = {}, key ) {
+  if ( state[key] ) {
+    return state[key];
+  }
+
+  return UNDEFINED;
+}
+
+/**
+ * Throw an error if the reducer processing the current action doesn't return
+ * a value
+ * @param {object} nextState The state object just returned by a reducer
+ */
```

Comment:
Comment could have `* @throws {Error} If the reducer processing the current action doesn't return a value.`

Reply:
Oh cool, my bad for not verifying if there was a `@throw` declaration